app/views/cloud_volume/_common_new_edit.html.haml

```diff
@@ -1,29 +1,80 @@
+.form-group{"ng-class" => "{'has-error': angularForm.storage_manager_id.$invalid}"}
+  %label.col-md-2.control-label
+    = _('Storage Manager')
+  .col-md-8
+    %select{"name"        => "storage_manager_id",
+            "ng-model"    => "vm.cloudVolumeModel.storage_manager_id",
+            "ng-options"  => "mgr.id as mgr.name for mgr in vm.storageManagers",
+            "ng-change"   => "storageManagerChanged(vm.cloudVolumeModel.storage_manager_id)",
+            "required"    => "",
```

Comment:
Even though you have the `required` here, for some reason it does not show up as a `Required` field for me.
<img width="1211" alt="screen shot 2017-04-12 at 12 22 35 pm" src="https://cloud.githubusercontent.com/assets/1538216/24976789/acc62710-1f7f-11e7-81fb-7a02f292e6a4.png">